halyard-deploy/src/main/java/com/netflix/spinnaker/halyard/deploy/spinnaker/v1/service/distributed/kubernetes/v2/KubernetesV2RedisService.java

```diff
@@ -43,7 +43,7 @@ public boolean runsOnJvm() {
   }
 
   public String getArtifactId(DeploymentConfiguration deploymentConfiguration) {
-    return "gcr.io/kubernetes-spinnaker/redis-cluster:v2";
+    return "us-docker.pkg.dev/spinnaker-community/redis-cluster:v2";
```

Comment:
Good catch - fixed. I put it in both `nightly` and `releases`, because it's own repo is probably overkill and would require more tooling changes to support.

No idea what it's used for - I assume it's the Redis we ship with.

Reply:
Okay, that sounds reasonable, thanks.